packages/fiori/src/WizardStep.ts

```diff
@@ -0,0 +1,150 @@
+	 *
+	 * <b>Note:</b> The text is displayed in the <code>ui5-wizard</code> navigation header.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.title
+	 * @defaultvalue ""
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property()
+	titleText!: string
+
+	/**
+	 * Defines the <code>subtitleText</code> of the step.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> the text is displayed in the <code>ui5-wizard</code> navigation header.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.subtitleText
+	 * @defaultvalue ""
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property()
+	subtitleText!: string
+
+	/**
+	 * Defines the <code>icon</code> of the step.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> The icon is displayed in the <code>ui5-wizard</code> navigation header.
+	 * <br><br>
+	 *
+	 * The SAP-icons font provides numerous options.
+	 * See all the available icons in the <ui5-link target="_blank" href="https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html" class="api-table-content-cell-link">Icon Explorer</ui5-link>.
+	 * @type {string}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.icon
+	 * @defaultvalue ""
+	 * @public
+	 */
+	@property()
+	icon!: string
+
+	/**
+	 * Defines if the step is <code>disabled</code>. When disabled the step is displayed,
+	 * but the user can't select the step by clicking or navigate to it with scrolling.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> Step can't be <code>selected</code> and <code>disabled</code> at the same time.
+	 * In this case the <code>selected</code> property would take precedence.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean
+
+	/**
+	 * Defines the step's <code>selected</code> state - the step that is currently active.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> Step can't be <code>selected</code> and <code>disabled</code> at the same time.
+	 * In this case the <code>selected</code> property would take precedence.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.selected
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	selected!: boolean
+
+	/**
+	 * When <code>branching</code> is enabled a dashed line would be displayed after the step,
+	 * meant to indicate that the next step is not yet known and depends on user choice in the current step.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> It is recommended to use <code>branching</code> on the last known step
+	 * and later add new steps when it becomes clear how the wizard flow should continue.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.fiori.WizardStep.prototype.branching
+
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	branching!: boolean
+
+	/**
+	 * Defines the step content.
+	 * @type {Node[]}
+	 * @slot
+	 * @public
+	 */
+
+	@slot({ type: HTMLElement, "default": true })
```

Comment:
default slot is not described with decorator if the component doesn't have managed slots. Also add @name attribute